src/NUnitFramework/tests/Internal/ParamNumberConversionsTests.cs

```diff
@@ -0,0 +1,73 @@
+using System;
```

Comment:
This is a great test class - thanks for adding it! Definitely shows the benefits of the change!